src/ts/Logger.ts

```diff
@@ -0,0 +1,48 @@
+export enum LogLevel {
+    Debug, Info, Warn, Error
+}
+
+export default class Logger {
+    static logLevel: LogLevel = LogLevel.Info;
```

Comment:
You'll notice I left in bit of logging. I could have deleted it all before opening the PR, but if we want to come back to modify or debug this code later, it's helpful if the logging is already in place and can be easily toggled.

There's a reason logging typically works like this basically everywhere...

This line is where the logLevel is currently set - so none of the existing log statements will output anything without modifying this.

I'm not familiar with the JS world. Is there a more conventional way to do conditional logging like this in JS?

Reply:
I've used https://github.com/pinojs/pino before but not sure it's warranted to bring that in here.

Reply:
Unless you have a clear preference, I'd prefer to just use this really minimal dumb thing for now rather than include an external dependency.

Reply:
We'll keep the dumb thing :)